src/uadk_rsa.c

```diff
@@ -604,161 +713,200 @@ static int uadk_wd_rsa_init(struct rsa_res_config *config,
 	return ret;
 }
 
-static void uadk_wd_rsa_uninit(void)
+static int uadk_e_rsa_init(void)
+{
+	struct uacce_dev *dev;
+	int ret;
+
+	if (g_rsa_res.pid != getpid()) {
+		pthread_spin_lock(&g_rsa_res.lock);
+		if (g_rsa_res.pid == getpid()) {
+			pthread_spin_unlock(&g_rsa_res.lock);
+		return 0;
+	}
```

Comment:
tab?

Reply:
> why change api?
> 
> * ```
>             uadk_destroy_rsa();
>   ```
> 
> * ```
>             uadk_e_destroy_rsa();
>   ```

Code inspectors think the uadk_ prefix is not clearly expressed, may represent uadk driver, and suggest using the uadk_e prefix to define the API, e is the abbr. of engine.

Reply:
> tab?

OK, I will make a change, tks.

Reply:
> tab?
OK, I will make a change, tks.